package.json

```diff
@@ -1,18 +1,21 @@
 {
-  "name": "d2l-button",
   "description": "Polymer-based web components and Sass mixins for D2L buttons",
-  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/button.git"
+  },
+  "name": "d2l-button",
   "scripts": {
     "build:sass": "node-sass --output-style expanded ./test/acceptance/button.css.scss ./test/acceptance/button.css",
     "predump": "rimraf test/acceptance/dumps",
     "dump": "concurrently -p name -n serve,galen -s first -k \"polymer serve\" \"npm run galen:local:dump\"",
-    "postinstall": "polymer install --variants",
     "galen:local:run": "d2l-galen test test/acceptance/button.test.js -g factory:local -s",
     "galen:local:dump": "d2l-galen dump test/acceptance/button.test.js -d test/acceptance/dumps -g factory:local -s",
     "galen:sauce:run": "sc-run -- d2l-galen test test/acceptance/button.test.js -g factory:sauce -s -i SAUCE_USERNAME SAUCE_ACCESS_KEY TRAVIS_REPO_SLUG TRAVIS_BUILD_NUMBER -- -p4",
-    "lint": "npm run lint:wc && npm run lint:html",
-    "lint:html": "eslint *.html demo/*.html test/*.html test/acceptance/*.html test/acceptance/*.js",
-    "lint:wc": "polymer lint -i d2l-button.html d2l-floating-buttons.html d2l-button-subtle.html d2l-button-icon.html",
+    "prelint": "npm run build:sass",
+    "lint": "npm run lint:wc && npm run lint:js",
+    "lint:js": "eslint .--ignore-path reports/** --ext .js,.html test/**/*.js test/**/*.html demo/**/*.js demo/**/*.html",
```

Comment:
Was the `--ignore-path reports/**` needed even though we have `reports` in the `.eslintignore` file?

Reply:
Whoops, it seems like I left this from my attempts to fix linting

The issue that I ran into was when running `polymer lint`, it seems to lint the reports directory despite adding `filesToIgnore: ["reports/**/*"]` to `polymer.json` (as seen [here](https://github.com/Brightspace/brightspace-activity-details-ui/blob/030f393421cb982d85029a36be3051eae2cd76da/polymer.json#L15))

Reply:
I've had lots of issues with running `polymer lint` locally, it always tries to lint things I don't want and then I get a memory error.  My solution has been to delete folders I don't want it to lint, but this isn't great :(

Reply:
Yeah I've also been doing that 😬 

I think the option to ignore paths was added, but from what I've seen there seems to have been some regression? https://github.com/Polymer/polymer-cli/issues/225

(I tried to downgrade `polymer-cli` but that didn't seem to work)